### ISSUE TITLE
Refactor style tasks

### ISSUE THREAD

**Issue by ragingwind** [yeoman/generator-polymer#153]
I think we can make a function for style tasks.

```javascript
var styleTask = function (stylesPath, srcs) {
  return gulp.src(srcs.map(function(src) {
      return 'app/' + stylesPath + src;
    }))
    .pipe($.changed(stylesPath, {extension: '.scss'}))
    .pipe($.rubySass({
        style: 'expanded',
        precision: 10
      })
      .on('error', console.error.bind(console))
    )
    .pipe(gulp.dest('.tmp/' + stylesPath))
    .pipe($.if('*.css', $.cssmin()))
    .pipe(gulp.dest('dist/' + stylesPath))
    .pipe($.size({title: stylesPath}));
}

gulp.task('styles', function () {
  return styleTask('styles', ['/**/*.css', '/*.scss']);
});

gulp.task('elements:style', function () {
  return styleTask('elements', ['/**/*.css', '/**/*.scss']);
});
```

**Comment by robdodson** [comment]
Yeah if you want to turn that into a PR that would be awesome :)

**Comment by uspetej** [comment]
Hi Rob,
sorry, I didn't quite understand your comment, me not having English as
 first language. :<)

I've been in IT long enough not to jump on a new tool or  program language
(last time I counted I think I have been programming in 19 programming
languages/environments)  before I have checked the whole development to
production path. Reason being I don't want to spend  endless hours
maintaining & supporting a System due to lack of initial knowledge how to
use the tools.

So what I am trying to figure out is the whole path from what tools to use
to production maintenance.
In this case I am sure it works, only I need to test and document before I
start.

Hope this makes sense....

best regards,
peter
